cg/services/order_validation_service/order_validation_service.py

```diff
@@ -1,8 +1,9 @@
-from workflows.tomte.validation_service import TomteOrderValidationService
+from abc import ABC, abstractmethod
 
+from cg.services.order_validation_service.models.validation_error import ValidationError
 
-class OrderValidationService:
-    """Service to orchestrate the order validation."""
 
-    def __init__(self, tomte_service: TomteOrderValidationService):
-        self.tomte_service = tomte_service
+class OrderValidationService(ABC):
+    @abstractmethod
+    def validate(self, order_json: str) -> ValidationError:
+        pass
```

Comment:
Just a question in regards to the endpoint the ordervalidationservice will be used.

How would that look like? Is the OrderValidationService called in that place, if so, should we not inject the other service here like we inject the field validator in the TomteValidationService further down? Or is this handled by the ABC?

Reply:
This class is actually an abstract class so it cannot be instantiated.
I'm not sure how it will look in the end, but the endpoint will somehow get a hold of a service exposing the validation logic